create_MyString_file.py

```diff
@@ -0,0 +1,30 @@
+class MyString:
+    def __init__(self,s):
+        self.string = s
+    
+    def __str__(self):
+        return self.string
+
+    def len(self):
+        count = 0
+        for char in self.string:
+            count += 1
+        return count
+
+    def __add__(self, other):
+        return self.string + other.string
+    
+    def substring(self,start,end,step):
```

Comment:
step always is 1, substring just take 2 param: `start`  `end`. like:
![image](https://user-images.githubusercontent.com/55494127/184926563-7ec03b38-385c-4e5c-be16-783b3b224a55.png)